src/base/bittorrent/tracker.cpp

```diff
@@ -149,6 +149,7 @@ using namespace BitTorrent;
 struct Tracker::TrackerAnnounceRequest
 {
     QHostAddress socketAddress;
+    quint16 requestPort = 0;
```

Comment:
I don't see why you need a field in this struct.

Reply:
Sure I could make it a variable because in this PR specifically it is only used in one scope. But in terms of design, and for future modifications, doesn't it make sense that the port number be part of this struct? The address is already there as well.

Reply:
>doesn't it make sense that the port number be part of this struct? 

No, at least not in this PR.

Reply:
Ok, since you insist, and I really don't have a specific more compelling reason to do it the other way, I'll change it.

Reply:
Fixed, and I also changed an instance of `ushort` to `quint16`, since that's what we use elsewhere.